hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/CrowdConfiguration.java

```diff
@@ -24,4 +30,10 @@ public RestTemplate restTemplate(@Qualifier("managementRequestFactory") ClientHt
         return new RestTemplate(clientHttpRequestFactory);
     }
 
+    @Bean
+    @Order(1)
```

Comment:
It could be usefull to depend on`CrowdMaintainer` order, eg when another maintainer would like to be ordered as `CrowdMaintainer.ORDER + 1`